### PR TITLE
Remove old admin panel warning

### LINKED ISSUE #4403: Is the "you should not be able to reach the administration panel" after setup warning still valid?
**Describe the topic**
In https://github.com/mhelmetag/strapi/blob/master/CONTRIBUTING.md, there's a section where it says "If you've followed the recommended setup, you should not be able to reach the administration panel at http://localhost:1337/admin." The thing is... I can view the admin page from that URL (after creating a root user) and `yarn develop` says I can too (from the screenshot below).

**Your suggestions for this topic**
Can we remove this from the contributing docs now? I believe it's no longer valid.

I can make a PR for that once I get the go ahead.

**Screenshots**
Screen Shot 2019-10-30 at 12 12 26 AM

**Code snippets**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by mhelmetag**
#### Description of what you did:

Removed an old warning about the admin panel needing more setup in the contrib doc and cleaned up surrounding bits.

Fixes #4403 

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,16 +88,7 @@ yarn develop
...
-
-**Start the administration panel server**
-
-```bash
-cd strapi/packages/strapi-admin
-yarn develop
-```
-
-The administration panel is available at http://localhost:4000/admin
+The administration panel is available at http://localhost:1337/admin
```
Comment (soupette) [inline CONTRIBUTING.md]: The built administration panel is available at http://localhost:1337/admin

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -88,16 +88,7 @@ yarn develop
 
 The server (API) is available at http://localhost:1337
 
-**WARNING** ⚠️ If you've followed the recommended setup, you should not be able to reach the administration panel at http://localhost:1337/admin.
-
-**Start the administration panel server**
```
Comment (soupette) [inline CONTRIBUTING.md]: We also need this part if you want to be able to modify the front-end files of the packages.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mhelmetag) [reply CONTRIBUTING.md]: 🤦‍♂ yep! I meant that haha

File: CONTRIBUTING.md (hunk 2 above)
Comment (mhelmetag) [reply CONTRIBUTING.md]: Ah! So you have to run `yarn develop` in both places?

Comment (soupette) [reply CONTRIBUTING.md]: Yes, we do when we need to modify the core packages

**Pull request merged**